addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -122,11 +122,18 @@ $o-we-zindex: $o-we-overlay-zindex + 1 !default;
             margin: 0 1px 0 -1px;
         }
 
+        .dropdown-menu .o_custom_gradient_btn {
+            width: 50% !important;
+        }
         .note-back-color-preview.dropup .dropdown-menu {
             // Highlight palette alignment
             left: -52px;
         }
     }
+    .link-custom-color-fill .dropdown-menu .o_custom_gradient_btn {
+        width: 50% !important;
+        box-sizing: border-box;
```

Comment:
Is not that the default in Odoo?

Reply:
In Odoo yes, but not in `.oe_toolbar` when opening the editor for text/highlight colors.